### DISCUSSION TITLE
Attribute error in utils type_conversion code in latest monai version 1.3.1

### DISCUSSION THREAD

**Discussion by xlar-sanjeet** [Project-MONAI/MONAI#7882] [General]
**Describe the bug**
I am getting this attribute error while trying to transform data for swin unetr code.

Environment
** Latest monai version 1.3.1**

Error --

File ~/.conda/envs/monai/lib/python3.10/site-packages/monai/utils/type_conversion.py:369, in convert_to_dst_type(src, dst, dtype, wrap_sequence, device, safe)
    368 output_type: Any
--> 369 if isinstance(dst, monai.data.MetaTensor):
    370     output_type = monai.data.MetaTensor

AttributeError: module 'monai' has no attribute 'data'

![monai_error](https://github.com/Project-MONAI/MONAI/assets/85297242/1db618c8-73df-4983-8b36-cf6a87bd9436)

**Comment by KumoLiu** [comment]
Hi @xlar-sanjeet, I can't reproduce the issue locally, could you please share a small script to reproduce the issue?
Such as:
```
 from monai.data import MetaTensor
```
Perhaps you can try to uninstall and reinstall the monai and ensure there is only one version exit.

Thanks!

**Comment by KumoLiu** [comment]
Will convert this one to discussion for now, feel free to create a new one, thanks.
